x-pack/legacy/plugins/siem/public/pages/case/components/push_to_service/index.tsx

```diff
@@ -69,25 +70,7 @@ export const usePushToService = ({
   const errorsMsg = useMemo(() => {
     let errors: Array<{ title: string; description: JSX.Element }> = [];
     if (actionLicense != null && !actionLicense.enabledInLicense) {
-      errors = [
-        ...errors,
-        {
-          title: i18n.PUSH_DISABLE_BY_LICENSE_TITLE,
-          description: (
-            <FormattedMessage
-              defaultMessage="To open cases in external systems, you must update your license to Platinum, start a free 30-day trial, or spin up a {link} on AWS, GCP, or Azure."
-              id="xpack.siem.case.caseView.pushToServiceDisableByLicenseDescription"
-              values={{
-                link: (
-                  <EuiLink href="https://www.elastic.co/cloud/" target="_blank">
-                    {i18n.LINK_CLOUD_DEPLOYMENT}
-                  </EuiLink>
-                ),
-              }}
-            />
-          ),
-        },
-      ];
+      errors = [...errors, getLicenseError()];
```

Comment:
Why don't we use the `getActionLicenseError` helper?

Reply:
because we want to keep the order the way it is